src/components/StoryCreationComponent.js

```diff
@@ -68,78 +52,122 @@ class StoryCreationComponent extends Component {
   };
 
   handleChangeTags = event => {
-    console.log(this.state.tagsField);
     let tag = event.target.value.toLowerCase();
     tag = tag.replace('\\', '');
     tag = tag.replace('*', '');
+    if (_.endsWith(tag, ' ') && tag.length == 1) {
+      this.setState({
+        tagsInputValue: ''
+      });
+      document.getElementById('tags-input-field').value = '';
+      return;
+    }
     if (_.endsWith(tag, ' ')) {
       let newArr = this.state.tagsField;
       newArr.push(tag);
       this.setState({
-        selectedAttributes: newArr
+        tagsField: newArr,
+        tagsInputValue: ''
       });
-      console.log(this.state.tagsField);
+
+      document.getElementById('tags-input-field').value = '';
     }
   };
 
-  handleChangeStory = event => {
-    let story = event.target.value.toLowerCase();
-    story = story.replace('\\', '');
-    story = story.replace('*', '');
-    this.setState({ storyField: story });
+  handleChangeStory = (content, delta, source, editor) => {
```

Comment:
This works, but it also counts new line character, fyi

Reply:
Added workaround. Can add more elegant solution later.